### PR TITLE
Tutorty

### PR DESCRIPTION
**Pull request by ChitvanRamani22**
## Related Issue 

- Info about the related issue 

Closes: #[issue number that will be closed through this PR]

### Describe the changes you've made

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement in the existing features
- [ ] Other

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Checklist:

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [ ] My changes generate no new warnings.
- [ ] I have verified/tested my code by running it locally.

## Additional Info (optional)
Any additional information you want to give

### CODE REVIEW

**Review by sagar-barapatre — APPROVED** [review]

File: src/components/card.jsx [hunk 1]
```diff
@@ -21,6 +21,8 @@ class cards extends Component {
             {this.props.user.title} {this.props.user.username}
           </Card.Title>
           <Card.Subtitle mb='2' text='muted'>
+            {this.props.user.tutor}
+            <br></br>
```
Comment (Namanl2001) [inline src/components/card.jsx:25]: - Shift tutor below subject info.


- change line 35 to  `</br>`

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
LGTM with some small nits, also resolve conflicts as the form style is updated to double coloumn.

**Comment by ChitvanRamani22** [comment]
@Namanl2001 i have made the required changes and pushed them .
Please do check it.

**Comment by Namanl2001** [comment]
IDK why you have force pushed but now I'm not able to pull and test your changes see [this](https://stackoverflow.com/questions/9813816/git-pull-after-forced-update). So either you can resolve this as mentioned in the above link or open a new pr or just simply attach the screenshots of the form in comments here, and please check them once by yourself so that we can merge this without any risk. You have already done some similar changes in the past so we can trust you 😄

**Comment by ChitvanRamani22** [comment]
I have attached all the screenshots.

Screenshot (456)
Screenshot (454)
Screenshot (453)
Screenshot (455)

**Comment by ChitvanRamani22** [comment]
@Namanl2001 everything is working fine  and  can also be seen from the ss attached .

**Review by Namanl2001 — APPROVED** [review]
Great! Thanks

**Pull request merged**